[2024-01-02T21:57:34.071+0100] {taskinstance.py:1149} INFO - Dependencies not met for <TaskInstance: dag_with_postgres_operator_v04.create_postgres_table scheduled__2023-07-16T00:00:00+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-01-02T21:57:34.072+0100] {local_task_job_runner.py:154} INFO - Task is not able to be run
